### PR TITLE
fix(PriceIdentifierUtils): Add correct price identifier decimals

### PR DESCRIPTION
**Pull request by smb2796**
**Motivation**

`PriceIdentiferUtils.js` needs to be updated to accommodate new price identifiers for synthetics that will be collateralized in USDC.

- uVOL-BTC-APR21: [umip-41](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-41.md)
- BTC-BASIS-3M/USDC: [umip-40](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-40.md#technical-specifications)
- BTC-BASIS-6M/USDC: [umip-40](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-40.md#technical-specifications) 
- ETH-BASIS-3M/USDC: [umip-40](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-40.md#technical-specifications)
- ETH-BASIS-6M/USDC: [umip-40](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-40.md#technical-specifications) 
- COMPUSDC-APR-FEB28/USDC: [umip-38](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-38.md#technical-specification)
- COMPUSDC-APR-MAR28/USDC: [umip-38](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-38.md#technical-specification)
- DSDUSD: [umip-37](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-37.md#technical-specification) 
- DEFI_PULSE_TOTAL_TVL: [umip-24](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-24.md#technical-specification)
- SUSHIUNI_TVL: [umip-24](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-24.md#technical-specification)

**Summary**

updated `PriceIdentiferUtils.js`

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [X]  Untested

### CODE REVIEW

File: packages/common/src/PriceIdentifierUtils.js [hunk 1]
```diff
@@ -11,7 +11,17 @@ const IDENTIFIER_NON_18_PRECISION = {
   "STABLESPREAD/BTC": 8,
   "ELASTIC_STABLESPREAD/USDC": 6,
   BCHNBTC: 8,
-  AMPLUSD: 6
+  AMPLUSD: 6,
+  "uVOL-BTC-APR21": 6,
```
Comment (nicholaspai) [inline packages/common/src/PriceIdentifierUtils.js:15]: Is uVOL collateralized in USDC?

**Review by nicholaspai — APPROVED** [review]
To confirm, all of these PID are intended to be collateralized in USDC or other 6 precision tokens?

File: packages/common/src/PriceIdentifierUtils.js (hunk 1 above)
Comment (smb2796) [reply packages/common/src/PriceIdentifierUtils.js:15]: Yep, it is.

**Comment by smb2796** [comment]
> To confirm, all of these PID are intended to be collateralized in USDC or other 6 precision tokens?

Yes, these are all intended to be used with USDC

**Pull request merged**